sdk/communication/Azure.Communication.PhoneNumbers/src/DiagnosticPhoneNumbersClientOptions.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core;
+
+namespace Azure.Communication.PhoneNumbers
+{
+    /// <summary> Represents phone number client options that are diagnostic. </summary>
+#pragma warning disable AZC0008 // ClientOptions should have a nested enum called ServiceVersion
+    public class DiagnosticPhoneNumbersClientOptions : PhoneNumbersClientOptions
+#pragma warning restore AZC0008 // ClientOptions should have a nested enum called ServiceVersion
+    {
+        private const ServiceVersion LatestVersion = ServiceVersion.V2021_03_07;
```

Comment:
Why do we need to have the version defined here again? Does not base constructor have default value for the version parameter?